src/groundStation/groundStation.py

```diff
@@ -46,14 +47,15 @@
 #     from ex2_ground_station_software.src.system import SystemValues
 #     import libcsp.build.libcsp_py3 as libcsp
 
+vals = SystemValues()
+apps = vals.APP_DICT
```

Comment:
Why did you take this out of the class? It should be in the class. If you need it externally it should be accesses by a getter